app/templates/gis-search-form.hbs

```diff
@@ -70,28 +70,88 @@
           <div class="ui loader"></div>
         </div>
         <div class="ui top attached fluid tabular menu">
-          {{#each tabSettings as |tab index|}}
-            <a
-              class="tab item {{if (eq index 0) "active"}}"
-              data-tab={{tab.tab}}
-              title={{t tab.title}}>
-              {{t tab.title}}
-            </a>
-          {{/each}}
+          <a
+            class="tab item active"
+            data-tab={{tabSettings.0.tab}}
+            title={{t tabSettings.0.title}}>
+            {{t tabSettings.0.title}}
+          </a>
+          <a
+            class="tab item"
+            data-tab={{tabSettings.1.tab}}
+            title={{t tabSettings.1.title}}>
+            {{t tabSettings.1.title}}
+          </a>
+        </div>
+        <div
+          class="ui bottom attached tab segment active"
+          data-tab={{tabSettings.0.tab}}>
+            <div class="ui segment">
```

Comment:
This toolbar segment must be top attached to table like bottom toolbar with paging.
Now it is visually devided and it looks strange.